role-make-version-changelog.sh

```diff
@@ -306,9 +306,12 @@ You have three options:
             cat "$rel_notes_file" CHANGELOG.md > .tmp-changelog
         fi
         mv .tmp-changelog CHANGELOG.md
-        git add CHANGELOG.md
+        # Expecting that this script is run from the cloned auto-maintenance repo
+        SCRIPT_DIR="$(dirname -- "$0")"
+        md2html_tool=pandoc sh "$SCRIPT_DIR"/md2html.sh README.md
+        git add CHANGELOG.md README.html
         { echo "docs(changelog): version $new_tag [citest skip]"; echo "";
-          echo "Create changelog update and release for version $new_tag"; } > .gitcommitmsg
+          echo "Update changelog and README.html for version $new_tag"; } > .gitcommitmsg
```

Comment:
I would also like to have a github action which creates README.html when README.md changes.
Note that the above might not change README.html, so the commit msg might be slightly misleading.